src/main/java/com/synopsys/integration/alert/provider/blackduck/validators/BlackDuckApiTokenValidator.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * blackduck-alert
+ *
+ * Copyright (c) 2020 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.alert.provider.blackduck.validators;
+
+import java.util.Optional;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.synopsys.integration.alert.provider.blackduck.BlackDuckProperties;
+import com.synopsys.integration.blackduck.exception.BlackDuckApiException;
+import com.synopsys.integration.blackduck.rest.BlackDuckHttpClient;
+import com.synopsys.integration.blackduck.service.BlackDuckServicesFactory;
+import com.synopsys.integration.blackduck.service.NotificationService;
+import com.synopsys.integration.exception.IntegrationException;
+import com.synopsys.integration.log.Slf4jIntLogger;
+import com.synopsys.integration.rest.RestConstants;
+
+public class BlackDuckApiTokenValidator {
+    private final BlackDuckProperties blackDuckProperties;
+    private final Logger logger = LoggerFactory.getLogger(BlackDuckApiTokenValidator.class);
+
+    public BlackDuckApiTokenValidator(BlackDuckProperties blackDuckProperties) {
+        this.blackDuckProperties = blackDuckProperties;
+    }
+
+    public boolean isApiTokenValid() {
+        Optional<BlackDuckHttpClient> optionalBlackDuckHttpClient = blackDuckProperties.createBlackDuckHttpClientAndLogErrors(logger);
+        return optionalBlackDuckHttpClient
+                   .map(this::isNotificationApiAllowed)
+                   .orElse(false);
+    }
+
+    private boolean isNotificationApiAllowed(BlackDuckHttpClient httpClient) {
+        boolean valid = true;
+        try {
+            BlackDuckServicesFactory blackDuckServicesFactory = blackDuckProperties.createBlackDuckServicesFactory(httpClient, new Slf4jIntLogger(logger));
+            NotificationService notificationService = blackDuckServicesFactory.createNotificationService();
+            notificationService.getLatestNotificationDate();
```

Comment:
Couldn't no notifications exist yet, but the config still be valid? A clean Black Duck instance could have this state.

Reply:
Same as the comment above, we don't really care about if there or aren't notifications. Even if it returned empty it would not give us a 403 forbidden which only occurs when the black duck user is lacking super user permissions

Reply:
Gotcha, so the status will still be 2xx?

Reply:
I could probably leave a comment in the code if that clears it up a bit more if you think that would be a good idae

Reply:
yes